spyder/plugins/editor/widgets/tests/test_warnings.py

```diff
@@ -89,10 +123,10 @@ def test_move_warnings(qtbot, lsp_codeeditor):
 
 
 @pytest.mark.slow
-@pytest.mark.first
+@pytest.mark.second
```

Comment:
@ccordoba12 what does this decorator refer to?

Reply:
It comes from `pytest-ordering`, a new dependency I added in PR #9175. It tells `pytest` to run these tests after the ones in `test_introspection`, which are marked with `@pytest.mark.first`. The thing is I noticed LSP tests tend to introduce odd problems among each other and with other tests if they are not run in this particular order (especially on Linux).

Reply:
Interesting! Thanks for the explanation